### PR TITLE
Update Authors

### PR DESCRIPTION
**Pull request by certik**
I've added all contributors listed at https://github.com/sympy/sympy-paper/graphs/contributors

They all seem to reasonably satisfy the criteria listed at https://github.com/sympy/sympy-paper/blob/112603990010549031a760dc69a0a385102b49bc/authors/README.md, so that @scopatz can merge it.

@asmeurer, 
@moorepants, 
@Upabjojr, 
@shivamvats, 
@aktech, 
@hargup, 
@scopatz, 
@Sumith1896, 
@leosartaj, 
@fredrik-johansson, 
@thilinarmtb, 
@fabianp,
please update your `email`, `institution` and `institution_address`. Just push into this branch if you can, or send a PR against it.

### CODE REVIEW

**Comment by certik** [comment]
@asmeurer, it looks like the latex template that we are using can't handle more than 9 authors. So I put the rest of the authors into a separate json file (that is currently ignored) for now. Let's still collect all the authors information and merge. We can figure out how to actually make them show in the pdf later.

**Comment by fabianp** [comment]
I made PR #103 as I don't have push privileges

File: authors/authors.extra.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        "email": "singhsartaj94@gmail.com",
+        "github_id": "leosartaj",
+        "institution": "Indian Institute of Technology (BHU)",
+        "institution_address": "Indian Institute of Technology (BHU), Varanasi, Uttar Pradesh, India - 221005",
+        "name": "Sartaj Singh"
+    },
+    {
+        "email": "??",
+        "github_id": "fredrik-johansson",
```
Comment (scopatz) [inline authors/authors.extra.json]: @fredrik-johansson

File: authors/authors.json [hunk 2]
```diff
@@ -1,12 +1,54 @@
...
+    {
         "email": "ondrej.certik@gmail.com",
         "github_id": "certik",
         "institution": "Los Alamos National Laboratory",
         "institution_address": "P.O. Box 1663, Los Alamos, New Mexico 87545, USA",
         "name": "Ond\u0159ej \u010cert\u00edk"
     },
     {
+        "email": "??",
+        "github_id": "moorepants",
```
Comment (scopatz) [inline authors/authors.json:28]: @moorepants

File: authors/authors.json [hunk 3]
```diff
@@ -1,12 +1,54 @@
...
+        "institution": "??",
+        "institution_address": "??",
+        "name": "Jason K. Moore"
+    },
+    {
+        "email": "??",
+        "github_id": "Upabjojr",
```
Comment (scopatz) [inline authors/authors.json:68]: @Upabjojr

**Comment by scopatz** [comment]
Just a few more people needed before this is mergable.

**Comment by certik** [comment]
@scopatz you broke it. The latex template can't handle more than 9 authors. You need to put yourself into the .extra file for now.

Also, I think you should finish your PR to sympy first. ;)

File: authors/authors.extra.json (hunk 1 above)
Comment (certik) [inline authors/authors.extra.json]: @fredrik-johansson can you please fill this in? So that we can merge.

File: authors/authors.json (hunk 3 above)
Comment (certik) [inline authors/authors.json:68]: @Upabjojr can you please fill this in?

**Comment by asmeurer** [comment]
Someone should make all the address formatting consistent before merging as well.

**Comment by asmeurer** [comment]
By the way, I checked, and the SIAM submission form appears to support as many authors as we have. So we'll just have to work with the journal to make sure they all get listed on the paper itself.

File: authors/authors.json (hunk 3 above)
Comment (certik) [inline authors/authors.json:68]: I just wrote Francesco an email to fill this in.

**Comment by certik** [comment]
Thanks @asmeurer for fixing it. Thanks @Upabjojr for updating your info. Thanks @isuruf for fixing it.

**Comment by certik** [comment]
@mattpap, I would love if you can get involved too. You can contact me privately for more info.

**Comment by certik** [comment]
@rpmuller, @rc, I just added you to authors. Would you mind checking your information to make sure it is correct?

**Comment by certik** [comment]
With this PR, we currently have 18 authors, and there are 19 contributors (i.e. have patches merged into master) to this repository, one of which is @gitter-badger. So we didn't forget anybody so far.

**Comment by rpmuller** [comment]
@certik Almost. "name" : "Richard P. Muller", "institution" : "Center for Computing Research, Sandia National Laboratories"

**Comment by certik** [comment]
@rpmuller thanks, I fixed it in f00775b5a9ca003495c6145773060e91e9432187.

**Comment by rc** [comment]
@certik Thanks, just a few tweaks: 

"institution": "New Technologies - Research Centre, University of West Bohemia"
"institution_address": "Univerzitní 8, 306 14 Plzeň, Czech Republic"

**Comment by certik** [comment]
@rc, fixed in cb4b3d54babc475bb7879965bfb025d66d64eba9, thanks!

**Comment by certik** [comment]
I fixed the SIAM template to allow more than 9 authors in ee4644a01bb29ce3bc2456034554385285613fdf. I didn't figure out a fix other than modifying the actual template. But at least it works now.

**Comment by certik** [comment]
I've added a new `np` field into the json, that lists the number of patches to sympy, and then I filter authors with patches > 0, and sort them using this key. It's an arbitrary measure, and we didn't even discuss yet how to order the authors, but ordering them by the number of patches contributed to sympy seems like a sensible first iteration. We can change that later.

@asmeurer, @moorepants, @scopatz would you mind reviewing my latest changes? If all is good, @scopatz can you please merge this? There are a few more authors coming, plus I expect there might be some changes in the `np` field, but we can just send a new PR for that.

**Comment by moorepants** [comment]
+1

**Comment by rc** [comment]
@certik when I saw `np` I thought numpy - what about using something more descriptive (`num_patches`, `num_commits`, ...)? The other fields are also whole words. Anyway, +1.

**Comment by aktech** [comment]
Or Simply name it as `commits` ?

**Comment by rc** [comment]
Yeah, `commits` says it all.

**Comment by certik** [comment]
sympy_commits would be even better.

**Comment by certik** [comment]
@rc, @aktech I renamed `np` to `sympy_commits` in 43c534a017c32294ce3cc9965ed175ddac258b98. Thanks for the tip.

**Comment by rc** [comment]
Thanks!

**Comment by certik** [comment]
Ok, I think @scopatz is too busy, so I am going to take responsibility here and merge it, so that we can move on. I think everybody who is included in this PR (with sympy_patches > 0) clearly passes the authorship criteria, so I think there is no issue with this PR.

**Comment by certik** [comment]
Later today I am going to submit PRs with the new authors that are not in yet, unless somebody beats me to it.

**Comment by asmeurer** [comment]
Thanks. I actually think you should be in charge of the authors, not @scopatz, since you are spending more time on it.

**Comment by certik** [comment]
Ok, I'll be in charge from now on.

**Pull request merged**